packages/manager/apps/key-management-service/src/pages/dashboard/index.tsx

```diff
@@ -39,9 +43,22 @@ export default function DashboardPage() {
     },
   ];
 
+  const breadcrumbItems: BreadcrumbItem[] = [
+    {
+      id: okmsId,
+      label: displayName || okmsId,
+      navigateTo: `/${okmsId}`,
+    },
```

Comment:
This item could be integrated directly in the useBreadcrumb hook since it is specific to the kms project and it is duplicated everytime

Reply:
`okmsId` comes from `useParam` and `displayName` comes from an API call, so i don't think it's a good idea to integrate it into the hook.